adding_species_manuscript.tex

```diff
@@ -536,9 +534,9 @@ \subsection*{Basic setup for chromosome-level simulations}
 These five categories of parameters are sufficient for generating simulations
 under neutral evolution. Such simulations are useful for a number of purposes,
 but they cannot be used to model the influence of natural selection on patterns of genetic variation.
-As mentioned above, it is a widely appreciated fact that linked selection modulates
+As mentioned above, it is a widely appreciated the fact that linked selection modulates
 patterns of variation within genomes.
-Therefore, its incorporation into simulations is crucial for many purposes.
+Therefore, its incorporation into simulations is crucial for simulating realistic genomes.
```

Comment:
I'm afraid that saying that it's crucial for realistic genomes will undercut the main message of this paper, which tries to set aside the issue of modeling selection. For this reason, 'many purposes' strikes a better balance, IMO.